Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -62,7 +62,7 @@ extension LogtoCore {
             "code": code,
             "code_verifier": codeVerifier,
             "client_id": clientId,
-            "resource": resource as Any,
+            "resource": resources as Any,
```

Comment:
Is it OK to use `Any` in swift?

Reply:
it is ok, since we are removing it if nil and it'll convert to json in the end